Jenkinsfile

```diff
@@ -180,31 +180,26 @@ pipeline {
                     steps {
                         sh '''rm -rf artifacts/ubuntu18.04/
                               mkdir -p artifacts/ubuntu18.04/
-                              mkdir -p _topdir
-                              rm -rf *.tar.gz
-                              rm -rf _topdir/BUILD/
-                              mkdir -p _topdir/BUILD
-                              : "${DEBEMAIL:="daos@daos.groups.io"}"
-                              : "${DEBFULLNAME:="daos-stack"}"
-                              : "${LANG:="C.UTF-8"}"
-                              : "${LC_ALL:="C.UTF-8"}"
+                              : "${DEBEMAIL:="$env.DAOS_EMAIL"}"
+                              : "${DEBFULLNAME:="$env.DAOS_FULLNAME"}"
                               export DEBEMAIL
                               export DEBFULLNAME
-                              export LANG
-                              export LC_ALL
-                              if make debs; then
-                                  ln -v _topdir/BUILD/*{.build,.changes,.deb,.dsc,.gz,.xz}  artifacts/ubuntu18.04/
-
-                                  pushd artifacts/ubuntu18.04/
-                                    dpkg-scanpackages . /dev/null | gzip -9c > Packages.gz
-                                  popd
-                              else
-                                  exit \${PIPESTATUS[0]}
-                              fi'''
-                        sh "cat _topdir/BUILD/*.build"
+                              make debs'''
                     }
                     post {
-                        always {
+                        success {
+                            sh '''ln -v \
+                                   _topdir/BUILD/*{.build,.changes,.deb,.dsc,.gz,.xz} \
+                                   artifacts/ubuntu18.04/
+                                  pushd artifacts/ubuntu18.04/
+                                    dpkg-scanpackages . /dev/null | \
+                                      gzip -9c > Packages.gz
+                                  popd'''
+                            archiveArtifacts artifacts: 'artifacts/ubuntu18.04/**'
+                        }
+                        failure {
+                            sh script: "cat _topdir/BUILD/*.build",
+                               returnStatus: true
```

Comment:
Curious what returning the status of the `cat` is achieving here.

Reply:
It is suppressing a failure of cat from causing an error in the failure handler if the file does not exist.
We are already in an fail condition at this state.